hummingbot/connector/exchange/trading212/trading212_api_user_stream_data_source.py

```
@@ -0,0 +1,323 @@
+                },
+                timestamp=asyncio.get_event_loop().time()
+            )
+            
+            # Emit to user stream tracker
+            if hasattr(self, '_user_stream_tracker'):
+                await self._user_stream_tracker._emit_position_close(position_message)
+                
+        except Exception as e:
+            self._logger.error(f"Error emitting position close: {e}")
+            
+    async def listen_for_user_stream(self, ev_loop: asyncio.AbstractEventLoop, output: asyncio.Queue):
```

Ruff (0.13.1)

Unused method argument: `ev_loop`

(ARG002)